public/app/features/explore/Explore.tsx

```diff
@@ -438,16 +439,17 @@ const getTimeRangeFromUrlMemoized = memoizeOne(getTimeRangeFromUrl);
 
 function mapStateToProps(state: StoreState, { exploreId }: ExploreProps): Partial<ExploreProps> {
   const explore = state.explore;
-  const { split, syncedTimes } = explore;
+  const urlQuery = state.location.query[exploreId] as string;
```

Comment:
We don't keep url duplicated inside explore state so just get it from `location` redux.